src/euphorie/client/browser/risk.py

```diff
@@ -261,18 +261,23 @@ def __call__(self):
         else:
             self._prepare_risk()
             if self.is_custom_risk:
```

Comment:
If my previous suggestion is accepted I would write:
```
if not self.is_custom_risk:
    return self.default_template()
```
and dedent the code one level